domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/UserRoleRequest.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.cmc.domain.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import javax.validation.constraints.NotEmpty;
+
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+@Builder
+@EqualsAndHashCode
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Please remove, we should avoid it as much as possible, unless really needed for the API that we do not control.